integration/reuse_layer_rebuild_test.go

```diff
@@ -194,6 +195,156 @@ func testReusingLayerRebuild(t *testing.T, context spec.G, it spec.S) {
 
 			Expect(secondImage.Buildpacks[6].Layers["assets"].SHA).To(Equal(firstImage.Buildpacks[6].Layers["assets"].SHA))
 		})
+
+		context("when the app has assets in extra destination paths", func() {
+			it.Before(func() {
+				appConfig, err := os.ReadFile(filepath.Join(source, "config", "application.rb"))
+				Expect(err).NotTo(HaveOccurred())
+
+				re, err := regexp.Compile(`# config.assets.prefix = "custom"`)
```

Comment:
This is where the linter is complaining. You can use `regexp.MustCompile` or just `Expect(err).NotTo(HaveOccurred())` as usual.